### PR TITLE
ConfigCI PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1880232](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1880232)

New cmdlet: Set-CIPolicySetting
Changes to 6 others

### CODE REVIEW

**Review by jgeurten — CHANGES_REQUESTED** [review]

**Review by get-itips — APPROVED** [review]

File: docset/winserver2022-ps/configci/New-CIPolicy.md [hunk 1]
```diff
@@ -407,6 +475,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MultiplePolicyFormat
+Indicates that this cmdlet should create a policy in multiple policy format as opposed to a single policy format. 
+Refer to [Create WDAC policies in Multiple Policy Format](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies#creating-wdac-policies-in-multiple-policy-format) for the difference between the policy formats.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/configci/New-CIPolicy.md:480]:
```suggestion
Refer to [Create WDAC policies in Multiple Policy Format](/windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies#creating-wdac-policies-in-multiple-policy-format) for the difference between the policy formats. 
```

File: docset/winserver2022-ps/configci/New-CIPolicy.md [hunk 2]
```diff
@@ -525,6 +609,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SpecificFileNameLevel
+Specifies the attribute of the file off which to base a file name rule. The -Level must be set to FileName for this option.
+Refer to [File Name Rules Info](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/configci/New-CIPolicy.md:614]:
```suggestion
Refer to [File Name Rules Info](/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```

File: docset/winserver2022-ps/configci/New-CIPolicyRule.md [hunk 3]
```diff
@@ -243,6 +246,43 @@ This set of commands finds a packaged application matching the specified name an
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AppID
+Specifies an app.
+This cmdlet creates per-app rules which control whether specific plug-ins, add-ins, and modules can run from specific apps.
+
+For more information, see [Use a Windows Defender Application Control policy to control specific plug-ins, add-ins, and modules](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-policy-to-control-specific-plug-ins-add-ins-and-modules).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/configci/New-CIPolicyRule.md:272]:
```suggestion
For more information, see [Use a Windows Defender Application Control policy to control specific plug-ins, add-ins, and modules](/windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-policy-to-control-specific-plug-ins-add-ins-and-modules).
```

File: docset/winserver2022-ps/configci/New-CIPolicyRule.md [hunk 4]
```diff
@@ -314,23 +354,22 @@ This cmdlet will not check whether the filepath string is a valid filepath.
...
 Position: Named
 Default value: None
-Accept pipeline input: True
-Accept wildcard characters: True
+Accept pipeline input: False
+Accept wildcard characters: False
 ```
 
 ### -Level
 Specifies the primary level of detail for generated rules. Refer to [WDAC File Rule Levels](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-file-rule-levels) for acceptable parameter values and descriptions.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/configci/New-CIPolicyRule.md:368]:
```suggestion
Specifies the primary level of detail for generated rules. Refer to [WDAC File Rule Levels](/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-file-rule-levels) for acceptable parameter values and descriptions.
```

File: docset/winserver2022-ps/configci/New-CIPolicyRule.md [hunk 5]
```diff
@@ -371,11 +410,11 @@ Accept wildcard characters: False
 ```
 
 ### -SpecificFileNameLevel
-Specifies the attribute of the file off which to base a file name rule. The -Level must be set to FileName for this option. 
-Refer to [File Name Rules Info](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values. 
+Specifies the attribute of the file off which to base a file name rule. The -Level must be set to FileName for this option.
+Refer to [File Name Rules Info](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/configci/New-CIPolicyRule.md:414]:
```suggestion
Refer to [File Name Rules Info](/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy looks good. However, I have suggested changing the links to site relative links, unless there is a reason not to. @TimShererWithAquent, @dariomws, @get-itips, @tseward. Many thanks.

**Review by dariomws — APPROVED** [review]
I'm ok after suggestions.

**Comment by TimShererWithAquent** [comment]
@JohanFreelancer9 Thanks - I rechecked the style guide and site relative links are correct, as you said. Accepted your changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
Hi @tiburd - I've accepted the changes from the review. This one should be ready to merge.

**Pull request merged**